python/lsst/daf/butler/formatters/matplotlib.py

```diff
@@ -38,8 +41,22 @@ class MatplotlibFormatter(FileFormatter):
 
     def _readFile(self, path: str, pytype: type[Any] | None = None) -> Any:
         # docstring inherited from FileFormatter._readFile
-        raise NotImplementedError(f"matplotlib figures cannot be read by the butler; path is {path}")
+        return plt.imread(path)
```

Comment:
Given that this is returning something unexpected (and effectively backwards from expectations) please add a comment saying that it's not returning a figure.